configs/base/package.json

```diff
@@ -24,19 +24,9 @@
     "index.js"
   ],
   "dependencies": {
-    "@mediamonks/eslint-config-base": "^2.0.0-alpha.4"
+    "@mediamonks/eslint-config-base": "^2.1.0"
   },
   "peerDependencies": {
-    "eslint": ">=8",
```

Comment:
I believe I added these so they can be installed with the `peerdeps` script (for when using `yarn`), since that doesn't do nested peerdeps. Are they not needed anymore?

Reply:
we can update that command to also include the base package's peer deps. This is what we did for the `react` and `vue` packages: https://github.com/mediamonks/eslint-config/tree/main/configs/react

Reply:
Sounds like a plan 👍